test/parallel/test-stdin-pipe-large.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+
+// https://github.com/nodejs/node/issues/5927
+
+if (process.argv[2] === 'child') {
+  process.stdin.pipe(process.stdout);
+  return;
+}
+
+const child = spawn(process.execPath, [__filename, 'child'], { stdio: 'pipe' });
+
+const expectedBytes = 1024 * 1024;
```

Comment:
Just out of curiosity, is there any significance to this number? Will this reproduce the issue on all operating systems?